frontend/src/component/playground/Playground/PlaygroundForm/renderOption.tsx

```diff
@@ -0,0 +1,23 @@
+import CheckBoxOutlineBlankIcon from '@mui/icons-material/CheckBoxOutlineBlank';
+import CheckBoxIcon from '@mui/icons-material/CheckBox';
+import React from 'react';
+import { Checkbox, styled } from '@mui/material';
+
+const SelectOptionCheckbox = styled(Checkbox)(({ theme }) => ({
+    marginRight: theme.spacing(0.4),
+}));
+
+export const renderOption = (
```

Comment:
we want to have checkboxes in the multi selection dropdown similar to API tokens for project selection